test/wpool_SUITE.erl

```diff
@@ -46,7 +55,65 @@ end_per_suite(Config) ->
 -spec overrun_handler(M) -> M.
 overrun_handler(M) -> overrun_handler ! {overrun, M}.
 
--spec overrun(config()) -> _.
+-spec too_much_overrun(config()) -> {comment, []}.
+too_much_overrun(_Config) ->
+  ct:comment("Receiving overruns here..."),
+  true = register(overrun_handler, self()),
+  {ok, PoolPid} =
+    wpool:start_sup_pool(
+      too_much_overrun,
+      [ {workers, 1}
```

Comment:
The code in the following (LINE, COL) locations has the same structure: (65, 7), (122, 7).